header/sync/sync_test.go

```diff
@@ -20,7 +20,7 @@ var blockTime = 30 * time.Second
 func TestSyncSimpleRequestingHead(t *testing.T) {
 	// this way we force local head of Syncer to expire, so it requests a new one from trusted peer
 	header.TrustingPeriod = time.Microsecond
-	requestSize = 13 // just some random number
+	header.RequestSize = 13 // just some random number
```

Comment:
we could fuzz this maybe, just an idea

Reply:
not in scope of this PR I thing

Reply:
But in general would be nice to add to testing ideas